torch/_refs/__init__.py

```diff
@@ -6542,7 +6542,11 @@ def _recursive_build(
 
     # seq can be a list of tensors
     seq = obj
-    return torch.stack([_recursive_build(scalarType, item) for item in seq])
+    return (
+        torch.tensor([])
```

Comment:
meh, don't call the stock torch.tensor here. Do something like torch.empty(0) instead

Reply:
Changed, thanks!